ddp-workspace/projects/ddp-testboston/src/app/components/welcome/welcome.component.html

```diff
@@ -40,6 +40,11 @@ <h2 class="center semi-bold" translate>HomePage.FaqSection.Title</h2>
                             <ng-container *ngFor="let paragraph of item.Paragraphs">
                                 <p class="question-block__text" [innerHTML]="paragraph"></p>
                             </ng-container>
+                            <ul class="faq-panel-list" *ngIf="item.Bullets && item.Bullets.length > 0">
+                                <li class="faq-panel-list__item" *ngFor="let bullet of item.Bullets">
+                                    <span class="question-block__text" [innerHTML]="bullet | translate"></span>
```

Comment:
I think I'll need help with the various HTML/SCSS/translate stuff..

Reply:
Do you mind, if I pull your branch tomorrow and make some changes?

Reply:
Rolled up all my changes into one PR, so hopefully easier to pick it up.